code/modules/multiz/structures.dm

```diff
@@ -68,6 +68,10 @@
 	if(target_ladder)
 		M.forceMove(get_turf(target_ladder))
 
+/obj/machinery/power/supermatter/attack_robot(var/mob/M)
```

Comment:
Wrong path?

Reply:
Copypaste is cruel, yet needed mistress. Fixed.